src/ext/openssl.cr

```diff
@@ -0,0 +1,84 @@
+require "openssl"
+
+class OpenSSL::X509::Certificate
+  def scram_signature
```

Comment:
Is this something scoped to PG or is something that would make sense to port into the std-lib also?